content/en/benchmark.md

```diff
@@ -0,0 +1,269 @@
+
+<!-- NumPy and Numba both use a similar type of compilation for ufuncs in manual looping resulting in the same speed.  Another thing that Numba lacks behind is that it does not support all functions of NumPy. There are functions in NumPy which does not hold up some of the optional arguments in nopython mode. It can implement linear algebra calls in the compiled functions but does not return any faster implementation. -->
+
+### Pythran
+
+> Pythran is an ahead of time compiler for a subset of the Python language, with a focus on scientific computing.
+
+<div style="text-align: right">Source: <a href="https://pythran.readthedocs.io/en/latest/#">Pythran's Website</a></div>
+
+Since the focus of Pythran was on accelerating Python and NumPy codes, its C++ API is the same as that of NumPy. Pythran also supports [Expression Templating](https://en.wikipedia.org/wiki/Expression_templates) and [SIMD](https://en.wikipedia.org/wiki/SIMD) instructions, which are its main advantages. It converts annotated Python modules into native Python modules, which are comparatively faster. But both have the same kind of interface.
+
+<!-- NumPy arrays in Cython should be stored in contiguous memory like C-style or Fortran to use Pythran in the backend. Here, the Pythran lacks behind. Another limitation is that the sequence of bytes of words must be the same as the targeted architecture to make Pythran work.-->
+
+## Source Code
+
+* The code is inspired by <a href = "https://github.com/paugier/nbabel">Pierre Augier's work on N-Body Problem</a>.
+* Visualization Code: <a href = "/benchmarks/python/plot.py">here</a>.
+
+<html>
+<head>
+<style>
+table, th, td {
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
+</head>
+ <table>
+  <tr>
+   <td><b>Algorithm & Source Code</b></td>
+   <td><b>Implementation Details</b></td>
+  </tr>
+  <tr>
+   <td><a href = "/benchmarks/python/optimized_numpy.py">NumPy</a></td>
+   <td>Vectorized Approach, Broadcasting Method, NumPy Arrays</td>
+  </tr>
+  <tr>
+   <td><a href = "/benchmarks/python/pure_python.py">Python</a></td>
+   <td>Standard Python Approach, Using List</td>
+  </tr>
+ <tr>
+  <td><a href = "/benchmarks/cpp/main.cpp">C++</a></td>
+  <td>C++ Implementation, GNU C++ Compiler</td>
+ </tr>
+ <tr>
+   <td><a href = "/benchmarks/python/compiled_methods.py">Numba</a></td>
+   <td>Just-In-time Compilation, Non-Vectorized Approach, Using Numba at the Backend via Transonic, NumPy Arrays</td>
+  </tr>
+ <tr>
+  <td><a href = "/benchmarks/python/compiled_methods.py">Pythran</a></td>
+  <td>Just-In-Time Compilation, Non-Vectorized Approach, Pythran at the Backend via Transonic, NumPy Arrays</td>
+ </tr>
+ </table>
+</html>
+
+## Results
+
+Table values represent the normalized time taken in seconds by each algorithm to run on the given datasets for $50$ number of iterations. The raw timing data can be downloaded from <a href = "benchmarks/data/table.csv">here</a>.
```

Comment:
Format to 80 columns, specify the normalization used in the table, and remove reduncancy
```suggestion
Table values represent the normalized time `time / n_particles` taken in
seconds by each algorithm to run on the given datasets for $50$ iterations. The
raw timing data can be downloaded from <a href =
"benchmarks/data/table.csv">here</a>.```

Reply:
Sure, I'll make the edits. Thanks!